snapcraft.yaml

```diff
@@ -201,6 +201,7 @@ parts:
     build-packages:
       - libportal-dev
       - libqrencode-dev
+      - libgstreamer1.0-dev
```

Comment:
Commit message is ambiguous since this is not staging it, however isn't gstreamer a dependency of gtk now? Isn't part of the SDK?

Reply:
Ops, yes they are, I see /usr/include/systemd/sd-daemon.h in the SDK. Maybe it was added in between? Let me re-try the build without that extra build-depends, will report back.

Reply:
Ugh, no they aren't, I conflated libsystemd-dev and libstreamer1.0-dev in my last comment, the previous one is really correct: The gstreamer header files are not in the SDK.

Reply:
@sergio-costas @jbicha do you think they should be?

Reply:
It's not clear to me what the scope of the Snap GNOME SDK is supposed to be. If it's supposed to match the Flatpak GNOME SDK, then yeah, [gstreamer](https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/master/elements/sdk/gst-plugin-gtk4.bst) is there.

Two snaps I touched last week that use gstreamer are Epiphany and [Shortwave](https://github.com/alexmurray/shortwave-snap/blob/main/snapcraft.yaml)

Reply:
We are staging gstreamer in the SDK, so I think that we must include the devel files.

Reply:
In fact... I wonder why aren't we staging the -dev versions here, when everything else are -dev versions...
![imagen](https://github.com/user-attachments/assets/87c6f257-cd30-4ec0-866b-57a6f6529b49)